### PR TITLE
Add initial work for CPU percentage

### PR DESCRIPTION
**Pull request by MihaiTabara**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -150,19 +153,30 @@ func collectStatsForWithError(proc *process.Process, withError bool) (*MozProces
...
 		// Three significant digits for the cpu times.
-		statistics.CPU = CPUTimesStat{
+		statistics.CPU = ProcCPUStat{
 			math.Round(cpu.User*1000) / 1000,
 			math.Round(cpu.System*1000) / 1000,
 			math.Round(cpu.Idle*1000) / 1000,
 			math.Round(cpu.Iowait*1000) / 1000,
-			math.Round(cpu.Steal*1000) / 1000}
+			math.Round(cpu.Steal*1000) / 1000,
+			cpu_percent}
```
Comment (MihaiTabara) [inline main.go]: For now I'm not round this up as I wanted to see the actual numbers after comma.

File: main.go [hunk 2]
```diff
@@ -21,13 +21,14 @@ import (
...
-	Iowait float64 `json:"iowait"`
-	Steal  float64 `json:"steal"`
+// ProcCPUStat is a more limited version of cpu.TimesStat to save storage space
+type ProcCPUStat struct {
+	User    float64 `json:"user"`
+	System  float64 `json:"system"`
+	Idle    float64 `json:"idle"`
+	Iowait  float64 `json:"iowait"`
+	Steal   float64 `json:"steal"`
+	Percent float64 `json:"percent"`
```
Comment (MihaiTabara) [inline main.go:31]: I debated whether to include this in a separate struct but made more sense to make the CPU collection as a `Proc`related item, as are the other things we're collecting. I'm open to whether to move it separately should you want.

File: main.go [hunk 3]
```diff
@@ -21,13 +21,14 @@ import (
...
 
-// CPUTimesStat is a more limited version of cpu.TimesStat to save storage space
-type CPUTimesStat struct {
-	User   float64 `json:"user"`
-	System float64 `json:"system"`
-	Idle   float64 `json:"idle"`
-	Iowait float64 `json:"iowait"`
-	Steal  float64 `json:"steal"`
+// ProcCPUStat is a more limited version of cpu.TimesStat to save storage space
+type ProcCPUStat struct {
```
Comment (MihaiTabara) [inline main.go:25]: I renamed this to include the `Proc` prefix to be consistent with other structs collecting per/process measurements.

Comment (srfraser) [reply main.go:25]: We can rename them if it's not clear, it's no trouble

**Comment by srfraser** [comment]
Needs `gofmt` running over it, but otherwise looks good. Thanks!

**Review by srfraser — APPROVED** [review]

**Pull request merged**